### PR TITLE
SE-1858

### PR DESCRIPTION
**Pull request by guy-martin**
- Add calibration mode
- Add support for Kalibr format
- Add support for calib.io format
- Define and support Bottlenose format for generic calibration
- Tested on stereo/mono on windows 10 and Ubuntu
- docs updated accordingly

### CODE REVIEW

File: distribution/requirements.txt [hunk 1]
```diff
@@ -2,4 +2,5 @@ PySide6>=6.4.3
 pyinstaller>=5.9.0
 Jinja2>=3.1.2
 numpy>=1.17.4
+pyyaml
```
Comment (treideme) [inline distribution/requirements.txt:5]: do a pip freeze and then pin this to your version like
pyyaml>=<what you see>

File: utility/requirements.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
 numpy>=1.17.4
-PySide6==6.5.0
\ No newline at end of file
+PySide6==6.5.0
+pyyaml
```
Comment (treideme) [inline utility/requirements.txt:3]: do a pip freeze and then pin this to your version like
pyyaml>=<what you see>

**Review by treideme — CHANGES_REQUESTED** [review]
See comments for changes...
Also please confirm that
(1) Uninstalled the previous edition from Windows and then reinstall and test on mono and stereo
(2) Tested on Linux
(3) Documentation updated to reflect this, including supported calibration formats

File: distribution/requirements.txt (hunk 1 above)
Comment (guy-martin) [reply distribution/requirements.txt]: Pinned version to 5.4.1

**Review by treideme — APPROVED** [review]

**Pull request merged**